front/app/modules/commercial/project_folders/citizen/components/ProjectFolderGoBackButton/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { memo, useCallback, FormEvent } from 'react';
+import clHistory from 'utils/cl-router/history';
+import { Button } from 'cl2-component-library';
+import useProjectFolder from '../../../hooks/useProjectFolder';
+import useLocale from 'hooks/useLocale';
+import useLocalize from 'hooks/useLocalize';
+import { isNilOrError } from 'utils/helperUtils';
+
+interface Props {
+  projectFolderId: string;
+  className?: string;
+}
+
+const GoBackButton = memo(({ projectFolderId, className }: Props) => {
+  const projectFolder = useProjectFolder({ projectFolderId });
+  const locale = useLocale();
+  const localize = useLocalize();
+
+  const onGoBack = useCallback(
```

Comment:
Nitpicking here, but as a rule of thumb we try to avoid using useCallback (so just a regular function instead) if it doesn't provide a substantial performance benefit (e.g. for rows in a long list it would make sense to avoid rerendering the entire list whenever a change in one of the rows occurs). In thise case I think it's fine to leave out the useCallback.

Reply:
Copied this part from the original [GoBackButton](https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/app/containers/IdeasShow/GoBackButton.tsx#L21)... in the `IdeasShow` container... should I also change it there?

Reply:
Yes, feel free to change it there as well